packages/cli/src/routeGeneration/templates/express.hbs

```diff
@@ -72,7 +72,14 @@ export function RegisterRoutes(app: express.Express) {
             }
 
             {{#if ../../iocModule}}
-            const controller: any = iocContainer.get<{{../name}}>({{../name}});
+            function isFunction(obj: any): obj is Function { return typeof obj === 'function'; }
```

Comment:
I've had to add this function to satisfy TypeScript's [type-guards](https://stackoverflow.com/a/50371820/7220776).

In the case of
```ts
if (typeof iocContainer === 'function' {
  container = iocContainer(request);
}
```
TypeScript can't determine `iocContainer === 'function'`, so will mark `iocContainer(request)` as an error in the case it's not a function.